### ISSUE TITLE
Beating vs. passing

### ISSUE THREAD

**Issue by czarkoff** [hyst329/OpenFool#37]
Game does not allow choice between passing and beating if the same card can be used for both.  Ie. if you have a trump card with the same rank as the one you are attacked with,  you can only use it to pass,  not to beat.

**Comment by hyst329** [comment]
Yes, that's a known issue. Maybe it should be selected via the short/long click (or tap/hold on mobile devices).
